packages/web-config-server/src/apiV1/utils/fetchIndicatorValues/checkAllDataElementsAreDhisIndicators.js

```diff
@@ -6,12 +6,16 @@
 // because the normal analytics/rawData.json endpoint does not return any data for indicators.
 // Will have to implement this properly with #tupaia-backlog/issues/2412
 // After that remove this file and anything related to it
+
 export const checkAllDataElementsAreDhisIndicators = async (models, dataElementCodes) => {
   const dataElements = await models.dataSource.find({
     code: dataElementCodes,
     type: 'dataElement',
   });
 
+  if (dataElementCodes.length !== dataElements.length) {
+    throw new Error('Could not find one of the data elements, please check each code');
+  }
```

Comment:
Might be nice just to list the requested data element codes here? Extra brownie points for filtering the requested list to find out which ones weren't found